wurst/systems/trade/TradeShipSupervisor.wurst

```diff
@@ -44,20 +45,28 @@ public class TradeShipSupervisor
         nextStop = firstStop.getNext()
         ships.put(ship, this)
         moveToNextStop()
-        EventListener.add(ship, EVENT_UNIT_ISSUED_TARGET_ORDER, -> onIssuedOrder())
-        EventListener.add(ship, EVENT_UNIT_ISSUED_POINT_ORDER, -> onIssuedOrder())
-        EventListener.add(ship, EVENT_UNIT_ISSUED_ORDER, -> onIssuedOrder())
+        EventListener.add(ship, EVENT_PLAYER_UNIT_ISSUED_TARGET_ORDER, -> onIssuedOrder())
+        EventListener.add(ship, EVENT_PLAYER_UNIT_ISSUED_POINT_ORDER, -> onIssuedOrder())
+        EventListener.add(ship, EVENT_PLAYER_UNIT_ISSUED_ORDER, -> onIssuedOrder())
```

Comment:
Which of these fires for what order? Do different ones fire for buying and actually issuing an order?

Reply:
Depends on target type. I suppose buying is either an immediate order or target order. Move is a point order. Stop is an immediate order.